scripts/ci/jobs/check-generated.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+
+ROOT="$(cd "$(dirname "${BASH_SOURCE[0]}")"/../../.. && pwd)"
+# shellcheck source=../../../scripts/ci/lib.sh
+source "$ROOT/scripts/ci/lib.sh"
+
+set -euo pipefail
+
+FAIL_FLAG="/tmp/fail"
+
+info 'Check .containerignore file is in sync with .dockerignore (If this fails, follow instructions in .containerignore to update it.)'
+function check-containerignore-is-in-sync() {
+    diff \
+        --unified \
+        --ignore-blank-lines \
+        <(grep -v -e '^#' .containerignore) \
+        <(grep -vF -e '/.git/' -e '/image/' -e '/qa-tests-backend/' .dockerignore) \
```

Comment:
This should be updated to what you changed between .dockerignore and .containerignore